nowcasting_dataset/data_sources/fake.py

```diff
@@ -172,32 +173,65 @@ def topographic_fake(batch_size, image_size_pixels):
     return Topographic(xr_dataset)
 
 
+def add_uk_centroid_osgb(x, y):
+    """Add a OSGB value to make coords in center of UK"""
+    lat = np.random.uniform(51, 55)
+    lon = np.random.uniform(-2.5, 1)
+
+    x_center, y_center = lat_lon_to_osgb(lat=lat, lon=lon)
+
+    x = x + x_center
+    y = y + y_center
+
+    return x, y
+
+
+def make_random_point_coords_osgb(size: int):
```

Comment:
_NIT_ I think I'd be tempted to rename them to `create_...` to keep this in line with `create_image_array` below